packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -323,7 +323,7 @@ class TextPainter {
     _needsLayout = true;
   }
 
-  /// {@macro flutter.dart:ui.text.TextWidthBasis}
```

Comment:
I didn't see the declaration of flutter.dart:ui.text.TextWidthBasis, so I think this will still be null. I checked the code, and it seems like TextWidthBasis does not passed to dart ui/text.dart, so I think we will probably provide the full documentation here.